spec/support/vcr.rb

```diff
@@ -0,0 +1,8 @@
+require 'vcr'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.